modules/central-services-public-config.adoc

```diff
@@ -158,6 +158,74 @@ Red Hat recommends that you do not specify a port number if you are exposing {pr
 | `central.exposure.route.enabled`
 | Use `true` to expose Central by using a route.
 This parameter is only available for {ocp} clusters.
+
+| `central.db.external`
+| *Tech Preview* Use `true` to specify that Central DB should not be deployed and that an external database will be used.
+
+| `central.db.source.connectionString`
+a| *Tech Preview* The connection string for Central to use to connect to the database. This is only used when `central.db.external` is set to true. The connection string must be in keyword/value format as described in https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING[PostgreSQL Docs].
```

Comment:
```suggestion
a| (Technology Preview) The connection string for Central to use to connect to the database. This is only used when `central.db.external` is set to true. The connection string must be in keyword/value format as described in https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING[PostgreSQL Docs].
```

Reply:
@kcarmichael08 Because this file is reference in multiple includes do I need to put it in additional resources for all of them or just install-central-config-options-ocp.adoc?

Reply:
Just the assembly file (install-central-config-options-ocp)